docs/product-assembly_showcase-application/src/main/asciidoc/main.adoc

```diff
@@ -0,0 +1,59 @@
+[id='assembly_case-management-showcase']
+
+include::product-shared-docs/document-attributes.adoc[]
+
+= Using the Showcase application for case management
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:imagesdir: topics/product-shared-docs/images
+:context: case-management-showcase
+include::product-shared-docs/author-group.adoc[]
+
+
+// Purpose statement for the assembly
+
+As a case worker or process administrator, you can use the Showcase application to manage and monitor case management applications while case work is carried out in {CENTRAL}.
+
+Case management differs from business process management (BPM) in that it focuses on the actual data being handled throughout the case and less on the sequence of steps taken to complete a goal. Case data is the most important piece of information in case handling, while business context and decision-making is in the hands of the human case worker.
+
+Use this document to install the Showcase application and start a case instance using the IT Orders sample case management project in {CENTRAL}. Use {CENTRAL} to complete the tasks required to complete an IT Orders case.
+
+
+.Prerequisites
+
+* {EAP_LONG} 7.1.0 is installed. For installation information, see the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.1/html-single/installation_guide/[_Red Hat JBoss EAP 7.1.0 Installation Guide_].
+* {PRODUCT} is installed and configured with {KIE_SERVER}. For more information see {URL_INSTALLING_ON_EAP}[_{INSTALLING_ON_EAP}_].
+* The `KieLoginModule` property is enabled in `standalone-full.xml`. This required to connect to the {KIE_SERVER}. For more information about configuring the {KIE_SERVER}, see {URL_PLANNING_INSTALL}[_{PLANNING_INSTALL}_].
```

Comment:
I would use something like _is setup_ or _is configured_ instead of _property is enabled_, as it isn't really just a property.